bootloader/src/pyi_launch.c

```diff
@@ -467,10 +469,12 @@ int pyi_launch_execute(ARCHIVE_STATUS *status)
     if (!*PI_Py_FileSystemDefaultEncoding) {
         char *saved_locale, *loc_codeset;
         saved_locale = strdup(setlocale(LC_CTYPE, NULL));
```

Comment:
That's the other weird part. "C" is the locale at the start of the program. If you want to enable country specific locale, you have to explicitly say `setlocale(LC_SOME_CATEGORY, "")` to enable [locale specific functionality in the functions in that category.](http://man7.org/linux/man-pages/man7/locale.7.html)

If you start up Python and run `import locale; locale.setlocale(locale.LC_ALL, None)`, you'll see the locale is initially "C". (Python even makes sure to set the locale back to "C" after its startup code.)